Tests/Functional/Hooks/InitializeStdWrapTest.php

```diff
@@ -25,6 +25,10 @@ class InitializeStdWrapTest extends FunctionalTestCase
 
     protected ExtensionConfiguration | MockObject $extensionConfigurationMock;
 
+    protected array $coreExtensionsToLoad = [
+        'typo3/cms-scheduler'
```

Comment:
Missing `,` at end of file. Why does php-cs-fixer does not tell you about? Is `Tests` directory in path declaration?